libs/features/mas/mocks/aem.js

```diff
@@ -1,29 +1,19 @@
 export async function withAem(originalFetch) {
     return async ({ pathname }) => {
-        if (/cf\/fragments\//.test(pathname)) {
+        const isPublish = /\/sites\/fragments\//.test(pathname);
+        const isAuthor = /cf\/fragments\//.test(pathname);
```

Comment:
maybe we can change '/sites/cf/fragments' & '/sites/fragments' just for 'author/fragments' and 'publish/fragments'?
or 'author' and 'edge'? it would be more clear

Reply:
these paths correspond to the real paths on AEM, I wouldn't touch it now.